Objects/Converters/ConverterRevit/ConverterRevitShared/ConverterRevit.Previews.cs

```diff
@@ -24,15 +26,19 @@ private ApplicationObject PreviewGeometry(Base @object)
       );
       return appObj;
     }
+#pragma warning disable CA1031 // Do not catch general exception types
     try
     {
       server = new DirectContext3DServer(meshes, Doc);
       appObj.Update(status: ApplicationObject.State.Created);
     }
     catch (Exception ex)
     {
+      // TODO : check if catch statement is necessary
+      SpeckleLog.Logger.LogDefaultError(ex);
       appObj.Update(status: ApplicationObject.State.Failed, logItem: ex.Message);
```

Comment:
I fail to see how this would ever throw unless `appObj` is null

Reply:
Presumably in DirectContext3DServer() ? if so what can that throw? Is this a top level handler?